packages/mui-material/src/StepIcon/StepIcon.d.ts

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
-import { InternalStandardProps as StandardProps } from '..';
+import { InternalStandardProps as StandardProps, SvgIconOwnProps } from '..';
 import { Theme } from '../styles';
 import { StepIconClasses } from './stepIconClasses';
 
 export interface StepIconProps
-  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+  // TODO v7: extend React.HTMLAttributes<SVGSVGElement> as svg is root component of StepIcon not div
+  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, 'color' | 'children'>,
```

Comment:
Why wait? Can't we fix this now? It's a bug fix, right?

Reply:
I agree with it being bug, but I'm expecting there will be considerable amount of users who are rely on wrong types. 

So I wasn't sure if we could fix this between major releases.